pkg/backend/diy/backend_test.go

```diff
@@ -37,9 +37,34 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/testing/diagtest"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/testing/iotest"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/tokens"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/cmdutil"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
 )
 
+//nolint:paralleltest // mutates global configuration
+func TestFullyQualifiedStackNames(t *testing.T) {
+	// Arrange
+	tmpDir := t.TempDir()
+	ctx := context.Background()
+	b, err := New(ctx, diagtest.LogSink(t), "file://"+filepath.ToSlash(tmpDir), nil)
+	assert.NoError(t, err)
+
+	expected := "organization/project-12345/stack-67890"
+	ref, err := b.ParseStackReference(expected)
+	require.NoError(t, err)
+
+	s, err := b.CreateStack(ctx, ref, "", nil)
+	require.NoError(t, err)
+
+	// Act
+	cmdutil.FullyQualifyStackNames = true
+	actual := s.Ref().String()
```

Comment:
Is it worth asserting what the actual result would be without `FullyQualifyStackNames` set?